modules/account-lib/src/coin/trx/iface.ts

```diff
@@ -55,7 +59,20 @@ export interface ValueFields {
 export interface TransferContract {
   parameter: Value;
 }
-
+export interface TriggerSmartContract {
+  parameter: ContractCallValue;
+  type?: string;
+}
+export interface ContractCallValue {
+  type_url?: string;
+  value: ContractCallValueFields;
+}
+export interface ContractCallValueFields {
+  data: string;
+  contract_address: string;
+  owner_address: string;
+}
+// /////////
```

Comment:
probably not needed (or use regions if I'm not getting why this is here)

Reply:
I've removed it